atlasdb-client/src/main/java/com/palantir/atlasdb/transaction/impl/ForwardingTransaction.java

```diff
@@ -170,4 +170,9 @@ public void setTransactionType(TransactionType transactionType) {
     public TransactionType getTransactionType() {
         return delegate().getTransactionType();
     }
+
+    @Override
+    public void disableReadWriteConflictChecking(TableReference tableRef) {
+        delegate().disableReadWriteConflictChecking(tableRef);
+    }
```

Comment:
non-actionable: Confirmed that this looks like it covers all of the layers transactions could be wrapped in